src/CoreFoundation/CFMutableString.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 Microsoft Corporation
+
+#if !COREBUILD
+
+using System;
+using System.Runtime.InteropServices;
+
+using Foundation;
+using ObjCRuntime;
+
+namespace CoreFoundation {
+	
+	public class CFMutableString : CFString {
+
+		public CFMutableString (IntPtr handle)
+			: this (handle, false)
+		{
+		}
+		
+		[Preserve (Conditional = true)]
+		protected CFMutableString (IntPtr handle, bool owns)
+			: base (handle, owns)
+		{
+		}
+
+		[DllImport (Constants.CoreFoundationLibrary)]
+		static extern unsafe /* CFMutableStringRef* */ IntPtr CFStringCreateMutable (/* CFAllocatorRef* */ IntPtr alloc, nint maxLength);
+
+		public CFMutableString (string @string = "", nint maxLength = default (nint))
+		{
+			// not really needed - but it's consistant with other .ctor
+			if (@string == null)
+				throw new ArgumentNullException (nameof (@string));
+			Handle = CFStringCreateMutable (IntPtr.Zero, maxLength);
+			if (@string != null)
+				CFStringAppendCharacters (Handle, @string, @string.Length);
+		}
+
+		[DllImport (Constants.CoreFoundationLibrary)]
+		static extern unsafe /* CFMutableStringRef* */ IntPtr CFStringCreateMutableCopy (/* CFAllocatorRef* */ IntPtr alloc, nint maxLength, /* CFStringRef* */ IntPtr theString);
+
+		public CFMutableString (CFString theString, nint maxLength = default (nint))
```

Comment:
`theString` -> `string` to match all the other API.

Reply:
That's the same names as the native API, which makes it easier to understand (when the documentation merger is used).